tools/c7n_oci/tests/conftest.py

```diff
@@ -18,13 +20,20 @@ class CustodianOCITesting(PyTestUtils, OCIFlightRecorder):
     """Pytest OCI Testing Fixture"""
 
 
-@pytest.fixture(scope="function")
+@pytest.fixture(scope=Scope.FUNCTION.value)
 def test(request):
     test_utils = CustodianOCITesting(request)
-    test_utils.addCleanup(reset_session_cache)
     return test_utils
 
 
+@pytest.fixture(scope=Scope.SESSION.value, autouse=True)
```

Comment:
the autouse/session scope seems to be leaking.

Reply:
Removed session scope.